plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryPlugin.kt

```diff
@@ -277,6 +279,10 @@ class SentryPlugin : Plugin<Project> {
                 }
             }
         }
+
+        if (extension.autoInstallation.enabled.get()) {
```

Comment:
I'm not entirely sure, but I believe this won't work. Have you verified that it works? The reason is that the value for `autoInstallation` is not available at configuration time

Reply:
No, haven't tried yet, thanks for the hint, I will try